stache-boilerplate/.travis.yml-sample

```diff
@@ -0,0 +1,12 @@
+language: node_js
+
+before_script:
+  - npm i -g grunt-cli blackbaud-stache-cli
+
+script:
+  - stache build --config=stache.yml,stache.prod.yml,stache.deploy.yml
```

Comment:
Building locally should be unaffected. Existing, internal sites should also be unaffected (would be worth testing this).